src/test/java/de/fearnixx/jeak/test/plugin/ControllerTestPlugin.java

```diff
@@ -15,34 +16,22 @@
 @JeakBotPlugin(id = "controllertestplugin")
 public class ControllerTestPlugin extends AbstractTestPlugin {
 
-    @Inject
-    @Config
-    public IConfig configRef;
-
-    @Inject
-    public IDataCache dataCache;
-
     @Inject
     IRestControllerService restControllerService;
 
 
     public ControllerTestPlugin() {
         super();
-        addTest("test_injected_configRef");
-        addTest("test_injected_dataCache");
     }
 
     @Listener
     public void onInitialize(IBotStateEvent.IInitializeEvent event) {
-        if (configRef != null) {
-            success("test_injected_configRef");
-        }
-
-        if (dataCache != null) {
-            success("test_injected_dataCache");
-        }
         restControllerService.registerController(TestController.class, new TestController());
-        restControllerService.registerController(SecondTestController.class, new SecondTestController());
+        try {
+            restControllerService.registerController(SecondTestController.class, new SecondTestController());
+        } catch (RegisterControllerException e) {
+            success("Should fail to register a duplicated controller");
```

Comment:
Test is never added :sweat_smile: Please remember to add the test in the plugin constructor.  
At the moment, it is also advisable to follow the convention of having descriptive test names but not actual descriptions. :slightly_smiling_face: